config/routes.rb

```diff
@@ -9,5 +9,13 @@
     end
   end
 
+  namespace :api do
+    namespace :v1 do
+      resources :posts, only: [:index], defaults: {format: :json} do
+        resources :comments, only: [:index, :create]
+      end
+    end
+  end
```

Comment:
- [ ] Please add routes and endpoint for authentication, both registration and login

* Login endpoint - **POST :** `/login`
* Registration endpoint - **POST :** `/register` or `/signup`

![image](https://user-images.githubusercontent.com/75126481/162589127-956b9820-9188-4a77-b302-32467680cbe8.png)